client/src/components/App.jsx

```diff
@@ -1,42 +1,94 @@
 import Homepage from "./Homepage";
-import { useEffect } from "react";
+import { useEffect, lazy, Suspense } from "react";
 import { useAnalytics } from "../hooks/useAnalytics";
 import NavBar from "./Navbar.jsx";
+import NavPanel from "./NavPanel.jsx";
 import Playground from "./Playground.jsx";
 import Contact from "./Contact.jsx";
 import { ToastContainer } from "react-toastify";
-import { Routes, Route } from "react-router-dom";
+import { Routes, Route, Navigate, useLocation } from "react-router-dom";
+
+const Dashboard = lazy(() => import("./Dashboard.jsx"));
+const About = lazy(() => import("./About.jsx"));
+const Blog = lazy(() => import("./Blog.jsx"));
+const Bookings = lazy(() => import("./Bookings.jsx"));
+const Payments = lazy(() => import("./Payments.jsx"));
+const Services = lazy(() => import("./Services.jsx"));
+const Tickets = lazy(() => import("./Tickets.jsx"));
+const Profile = lazy(() => import("./Profile.jsx"));
+const Resources = lazy(() => import("./Resources.jsx"));
+const Users = lazy(() => import("./Users.jsx"));
 
 function App() {
   const { logEvent } = useAnalytics();
+  const location = useLocation();
+
+  // Routes that should show the regular NavBar (public routes)
+  const publicRoutes = ['/', '/home', '/playground', '/contact'];
+  
+  // Routes that should show NavPanel (management routes)  
+  const managementRoutes = ['/dashboard','/bookings', '/resources', '/profile', '/payments', '/blog', '/services', '/about', '/users', '/tickets'];
+  
+  const isPublicRoute = publicRoutes.includes(location.pathname);
+  const isManagementRoute = managementRoutes.includes(location.pathname);
 
   useEffect(() => {
     logEvent("screen_view", {
-      firebase_screen: "Home Page",
+      firebase_screen: location.pathname || "/",
       firebase_screen_class: "App",
     });
-  }, [logEvent]);
-  return (
-    <>
-      <NavBar />
-      <Routes>
-        <Route path="/playground" element={<Playground />} />
-        <Route path="/contact" element={<Contact />} />
-        <Route path="/" element={<Homepage />} />
-      </Routes>
-      {/*Reusable toast*/}
-      <ToastContainer
-        position="top-right"
-        autoClose={5000}
-        hideProgressBar={false}
-        newestOnTop={false}
-        closeOnClick
-        rtl={false}
-        pauseOnFocusLoss
-        draggable
-        pauseOnHover
-      />
-    </>
-  );
+  }, [logEvent, location.pathname]);
+    return (
+        <>
+          {/* Public routes - show NavBar */}
+          {isPublicRoute && <NavBar />}
+
+          {/* Management routes - show NavPanel layout */}
+          {isManagementRoute ? (
+            <div className="d-flex vh-100">
+             <NavPanel />
+              <main role="main" className="flex-fill bg-light overflow-auto">
+                <Suspense fallback={<div className="p-4">Loading…</div>}>
+                <Routes>
+                  <Route path="/dashboard/*" element={<Dashboard />} />
```

Comment:
I suggest removing the actual component files and simply replace with a text. This will avoid needing to add coverage yourself on files that you do not need to work on. You can replace the components with just text since CI action will not allow you to merge the feature without enough coverage.
```suggestion
                  <Route path="/dashboard/*" element={<p>Dashboard</p>} />
```

Reply:
okay sure noted and thanks 
I've corrected it